### PR TITLE
[CU-2dve0nb] docs(Technical stack overview): Replaced old page with new design

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2dve0nb

## Checklist

~~- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~~
~~- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~~

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2dve0nb mdbook changes](https://app.clickup.com/t/2dve0nb)

File: book/src/products/technical-stack-overview.md [hunk 1]
```diff
@@ -1,30 +1,60 @@
-# Technical Stack Overview
+# **[Technical Stack Overview](https://dali.devnets.composablefinance.ninja/products/technical-stack-overview.html#technical-stack-overview)**
+
 
 ---
 
-## Simplicity at Our Core: Our Guiding Principle
+
+## **[Simplicity at Our Core: Our Guiding Principle](https://dali.devnets.composablefinance.ninja/products/technical-stack-overview.html#simplicity-at-our-core-our-guiding-principle)**
```
Comment (cor) [inline book/src/products/technical-stack-overview.md:7]: why did you make these headings links?

File: book/src/products/technical-stack-overview.md [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
 
 Unnecessary complexity in many cases causes end-user friction and barricades developers from either expanding innovation or accessing it. To this end, we believe that sophisticated infrastructures must be rooted in simple and easy to use interfaces to propel efficiency and enable builders to ship confidently and faster. That is why at Composable Finance ‘simplicity’ is engrained in the DNA of our technologies and development ethos. Our purpose is to ensure pioneers get the most out of our whole host of products in the Composable ecosystem, and thus create exceptional experiences for the end user.
 
 Upon the launch of our technology, our team and community of developers are committed to furthering the Composable tech stack through continual upgrades to user experience, exploring and attacking new opportunities to strive to push DeFi towards ultimate ubiquity.
 
-![Composable Technical Stack](./composable-technical-stack.png)
-*The Composable Technical Stack unites the presently siloed blockchain and DeFi industries, creating a united ecosystem that makes the user experience both powerful and streamlined.*
+
+## **Our Current Technical Stack**
```
Comment (cor) [inline book/src/products/technical-stack-overview.md:16]: You should not make `##`'s bold with `**`, as that is done automatically

File: book/src/products/technical-stack-overview.md [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+
+DeFi developers are able to leverage the existing suite of foundational pallets, such as Mosaic, Cubic, Apollo, Centauri, and more, to rapidly design, build, and launch interoperable products. Each foundational pallet provides unique and necessary DeFi functionality which can now be stacked and incorporated to make functions modular. Our infrastructure enabled ease of pallet composition and the immense flexibility of our tech stack provides each new innovation with **modular composability**.
+
+![technical_stack_overview](composable-technical-stack.jpg) 
+*The core components of the Composable Technical Stack. Similar to how Port Control Protocol became an essential networking fabric for the Internet, Composable’s vision is to become the entryway and fabric for interactions, transfers, and communication cross-ecosystem. The result is a Port Control Protocol-like system for blockchains.*
+
+### **The Composable Application Layer**
```
Comment (cor) [inline book/src/products/technical-stack-overview.md:23]: You should not make `###`'s bold with `**`, as that is done automatically

**Review by cor — DISMISSED** [review]
Looks good, but needs some changes

File: book/src/products/technical-stack-overview.md [hunk 4]
```diff
@@ -1,30 +1,60 @@
...
+
+
+### The Composable Application Layer consists of applications on both our [Kusama Parachain (Picasso)](https://dali.devnets.composablefinance.ninja/products/the-picasso-parachain.html) and our [Polkadot Parachain (Composable)](https://dali.devnets.composablefinance.ninja/products/the-composable-parachain.html). We will host a suite of applications as pallets that encompass all major DeFi primitives and ecosystem components on the Picasso parachain, our custom-built L1 on Kusama.
```
Comment (PoisonPhang) [inline book/src/products/technical-stack-overview.md:26]: This should be a paragraph (no whole line markdown modifiers) not a heading 3

File: book/src/products/technical-stack-overview.md [hunk 5]
```diff
@@ -1,30 +1,60 @@
...
+### **The Composable Application Layer**
+
+
+### The Composable Application Layer consists of applications on both our [Kusama Parachain (Picasso)](https://dali.devnets.composablefinance.ninja/products/the-picasso-parachain.html) and our [Polkadot Parachain (Composable)](https://dali.devnets.composablefinance.ninja/products/the-composable-parachain.html). We will host a suite of applications as pallets that encompass all major DeFi primitives and ecosystem components on the Picasso parachain, our custom-built L1 on Kusama. 
+
+
+### **The Composable Cross-Chain Virtual Machine**
+
+
+### The [Composable Cross-Chain Virtual Machine (XCVM)](https://dali.devnets.composablefinance.ninja/products/cross-chain-virtual-machine.html) is a top-level orchestration layer for deploying protocols that can communicate across multiple chains at once. It handles the complexity of routing to different networks so that developers can focus on while architecting cross-chain native smart contracts and protocols..
```
Comment (PoisonPhang) [inline book/src/products/technical-stack-overview.md:32]: Should be paragraph, not heading 3

File: book/src/products/technical-stack-overview.md [hunk 6]
```diff
@@ -1,30 +1,60 @@
...
+
+
+### **The Composable Routing Layer**
+
+
+### The [Routing Layer](https://dali.devnets.composablefinance.ninja/products/routing-layer.html) is our pathway execution layer that assesses all of the possibilities for a given action (e.g taking out a loan of 1,000 USDC) across all potential layers and chains and selects the optimal pathway for a user. This layer will be crypto-economically secured, with incentives provided for actors to properly select the best routes for user actions. Thus, this layer will act as a function aggregator, providing optimal services to users without them having to scour the entire expanse of the DeFi space themselves for the most promising opportunities.
```
Comment (PoisonPhang) [inline book/src/products/technical-stack-overview.md:38]: Should be paragraph, not heading 3

**Review by PoisonPhang — CHANGES_REQUESTED** [review]

**Review by PoisonPhang — DISMISSED** [review]
LGTM! To fix CI, you might need to rebase on main. 
Feel free to message me on Slack if you'd like any advice on rebasing or need a new approval afterward.

**Review by 0xBrainjar2 — COMMENTED** [review]
Please use new tech stack image jafar has

**Review by PoisonPhang — APPROVED** [review]

**Review by PoisonPhang — APPROVED** [review]

**Comment by KaiserKarel** [comment]
@Martin-Kiss please rebase

**Comment by Martin-Kiss** [comment]
This PR can be closed once #1180 has been merged.

**Comment by Martin-Kiss** [comment]
Redundant PR, closed.

**Pull request closed without merging**